Configuration/NativeMethods.cs

```diff
@@ -64,7 +64,9 @@ private struct AssemblyInfo
         public static string GetAssemblyPath(string name)
         {
             if (name == null)
-                throw new ArgumentNullException("name");
+                throw new ArgumentNullException("name");
+              
+            if( Type.GetType( "Mono.Runtime" ) != null ) return null;
 
```

Comment:
Fixed in this checkin.
https://github.com/jonsequitur/Its.Configuration/commit/c7ddef9bda2735aa7421f6cd4ad8732e6ca39491